data/css/buffet.scss

```diff
@@ -119,6 +120,15 @@ $title-alt-font: 'Lato Thin' !default;
         background-color: $separator-color;
     }
 
+    .CardList {
+        &__thumbnail {
+            min-height: 200px;
+        }
+        .composite .CardList__thumbnail {
```

Comment:
Is not that. is about how scss gets translated/expanded. If you don't use the ```&``` there you will get:

```  .search-results .CardList .composite .CardList__thumbnail ```

which doesn't work (and not what we want either), vs:

```  .composite .search-results .CardList .CardList__thumbnail ```

that you would get by using ```&```.